api/src/validation.php

```diff
@@ -39,8 +39,16 @@
             'email' => v::optional(v::notEmpty()->length(1, 256)::email()),
             'isAdmin' => v::optional(v::boolType())
         ],
+        'updateUser' => [
+            'username' => v::optional(v::notEmpty()->noWhitespace()->length(1, 101)),
+            'lastname' => v::optional(v::notEmpty()->noWhitespace()->length(1, 51)),
+            'firstname' => v::optional(v::notEmpty()->noWhitespace()->length(1, 51)),
+            'password' => v::optional(v::notEmpty()->length(1, 256)),
```

Comment:
Isn't the min. password length 8?

Reply:
To be honest I just copied the validation of create user.

But you're right.